components/Form/Pc.vue

```diff
@@ -0,0 +1,200 @@
+    },
+    motherBoard() {
+      const motherBoardList = this.$store.getters['motherboard/motherBoard']
+      return motherBoardList.map((motherBoard: MotherBoard) => {
+        return {
+          label: motherBoard.name,
+          value: motherBoard.id
+        }
+      })
+    },
+    ram() {
+      const ramList = this.$store.getters['ram/ram']
+      return ramList.map((ram: Ram) => {
+        return {
+          label: ram.name,
+          value: ram.id
+        }
+      })
+    },
+    gpu() {
+      const gpuList = this.$store.getters['gpu/gpu']
+      return gpuList.map((gpu: Gpu) => {
+        return {
+          label: gpu.name,
+          value: gpu.id
+        }
+      })
+    },
+    hardDisc() {
+      const hardDiscList = this.$store.getters['hard-disc/disc']
+      return hardDiscList.map((hardDisc: HardDisc) => {
+        return {
+          label: hardDisc.name,
+          value: hardDisc.id
+        }
+      })
+    },
+    ssd() {
+      const ssdList = this.$store.getters['ssd/ssd']
+      return ssdList.map((ssd: Ssd) => {
+        return {
+          label: ssd.name,
+          value: ssd.id
+        }
+      })
+    },
+    casePc() {
+      const casePcList = this.$store.getters['case/case']
+      return casePcList.map((casePc: Case) => {
+        return {
+          label: casePc.name,
+          value: casePc.id
+        }
+      })
+    },
+    psu() {
+      const psuList = this.$store.getters['psu/psu']
+      return psuList.map((psu: Psu) => {
+        return {
+          label: psu.name,
+          value: psu.id
+        }
+      })
+    },
+
+  },
+ watch: {
+    data: {
+      deep: true,
+      handler(newValue, oldValue) {
+       const cpu = this.$store.getters['cpu/cpuById'](newValue.cpu?.value)
+       const cooller = this.$store.getters['cooller/coollerById'](newValue.cooller?.value)
+       const motherBoard = this.$store.getters['motherboard/motherBoardById'](newValue.motherBoard?.value)
+       const ram = this.$store.getters['ram/ramById'](newValue.ram?.value)
+       const gpu = this.$store.getters['gpu/gpuById'](newValue.gpu?.value)
+       const hardDisc = this.$store.getters['hard-disc/discById'] (newValue.hardDisc?.value)
+       const ssd = this.$store.getters['ssd/ssdById'](newValue.ssd?.value)
+       const casePc = this.$store.getters['case/caseById'](newValue.casePc?.value)
+       const psu = this.$store.getters['psu/psuById'](newValue.psu?.value)
+
+       const price = Number(cpu?.price || 0) + Number(cooller?.price || 0) + Number(motherBoard?.price || 0)+ Number(ram?.price || 0) + Number(gpu?.price || 0)+ Number(hardDisc?.price || 0) + Number(ssd?.price || 0) + Number(casePc?.price || 0) + Number(psu?.price || 0)
+
+       const data = {
+        cpu: cpu,
+        cooller: cooller,
+        motherboard: motherBoard,
+        ram: ram,
+        gpu: gpu,
+        hard_disc: hardDisc,
+        ssd: ssd,
+        casePc: casePc,
+        psu: psu,
+        price: price
+       }
```

Comment:
const data = {
  cpu: this.$store.getters['cpu/cpuById'](newValue.cpu?.value),
  cooller: this.$store.getters['cooller/coollerById'](newValue.cooller?.value),
  motherboard: this.$store.getters['motherboard/motherBoardById'](newValue.motherBoard?.value),
  ram: this.$store.getters['ram/ramById'](newValue.ram?.value),
  gpu: this.$store.getters['gpu/gpuById'](newValue.gpu?.value),
  hard_disc: this.$store.getters['hard-disc/discById'] (newValue.hardDisc?.value),
  ssd: this.$store.getters['ssd/ssdById'](newValue.ssd?.value),
  casePc: this.$store.getters['case/caseById'](newValue.casePc?.value),
  psu: this.$store.getters['psu/psuById'](newValue.psu?.value),
  price: 0,
}

      let price = 0

      Object.values(data).forEach(item => {
        price += Number(item?.price || 0)
      })

      data.price = price

      this.$store.commit('pc/addBuildPc', data)